SparkPost/2web2ui
# FE-1165 - The tooltip in the Engagement Recency table wraps in an unusual way

[FE-1165](https://sparkpost.atlassian.net/browse/FE-1165)

### What Changed
- Removes use of the `style` attribute in the `TooltipMetric` component, replaces styles with `styled-components`
- Adds `tooltipWidth` support to the `Sparkline` component in order to address the bug

### How To Test
- Log in with a 108 account and go to `/signals/engagement` - verify the tooltip content no longer wraps when hovering over the line chart in the table
- Verify the tooltip in both the OG and the Hibana themes
- Verify other instances of the `<TooltipMetric />` component still render correctly in the OG and Hibana themes

### To Do
